# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1306987214 25200
#      Wed Jun 01 21:00:14 2011 -0700
# Branch stable
# Node ID fb93b94dfc8260b9e02dd26033f191306a26345b
# Parent  ee4775d04d7f2dd0a9488fdb3e0aa7cd1a75fcf4
Add rectangle to new functions list in NEWS.

* NEWS: Add rectangle to new functions list in NEWS.
* unimplemented.m: Remove rectangle from list.
* rectangle.m: Use double spaces for sentence breaks in Texinfo

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -345,25 +345,25 @@ Summary of important user-visible change
 
       format short eng
       format long eng
 
     Both display numbers in engineering notation, i.e., mantissa +
     exponent where the exponent is a multiple of 3.
 
  ** The following functions are new in Octave 3.4:
-      accumdim   divergence   merge        ppjumps    textread
-      bitpack    erfcx        nfields      pqpnonneg  uigetdir
-      bitunpack  fileread     nth_element  quadcc     uigetfile
-      blkmm      fminbnd      onCleanup    randi      uiputfile
-      cbrt       fskipl       pbaspect     repelems   uimenu
-      curl       ifelse       pie3         reset      whitebg
-      chop       ishermitian  powerset     rsf2csf 
-      colstyle   isindex      ppder        saveas  
-      daspect    luupdate     ppint        strread 
+      accumdim   divergence   merge        ppjumps    strread 
+      bitpack    erfcx        nfields      pqpnonneg  textread
+      bitunpack  fileread     nth_element  quadcc     uigetdir
+      blkmm      fminbnd      onCleanup    randi      uigetfile
+      cbrt       fskipl       pbaspect     rectangle  uiputfile
+      curl       ifelse       pie3         repelems   uimenu
+      chop       ishermitian  powerset     reset      whitebg  
+      colstyle   isindex      ppder        rsf2csf  
+      daspect    luupdate     ppint        saveas  
 
  ** Using the image function to view images with external programs such
     as display, xv, and xloadimage is no longer supported.  The
     image_viewer function has also been removed.
 
  ** The behavior of struct assignments to non-struct values has been
     changed.  Previously, it was possible to overwrite an arbitrary
     value:
diff --git a/scripts/miscellaneous/unimplemented.m b/scripts/miscellaneous/unimplemented.m
--- a/scripts/miscellaneous/unimplemented.m
+++ b/scripts/miscellaneous/unimplemented.m
@@ -324,17 +324,16 @@ function list = missing_functions ()
   "profsave",
   "propedit",
   "propertyeditor",
   "publish",
   "qmr",
   "quad2d",
   "questdlg",
   "rbbox",
-  "rectangle",
   "recycle",
   "reducepatch",
   "reducevolume",
   "resample",
   "reset",
   "rgbplot",
   "rmpref",
   "root",
diff --git a/scripts/plot/rectangle.m b/scripts/plot/rectangle.m
--- a/scripts/plot/rectangle.m
+++ b/scripts/plot/rectangle.m
@@ -12,26 +12,26 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Function File} {} rectangle ()
+## @deftypefn  {Function File} {} rectangle ()
 ## @deftypefnx {Function File} {} rectangle (@dots{}, "Position", @var{pos})
 ## @deftypefnx {Function File} {} rectangle (@dots{}, "Curvature", @var{curv})
 ## @deftypefnx {Function File} {} rectangle (@dots{}, "EdgeColor", @var{ec})
 ## @deftypefnx {Function File} {} rectangle (@dots{}, "FaceColor", @var{fc})
 ## @deftypefnx {Function File} {@var{h} =} rectangle (@dots{})
 ##
-## Draw rectangular patch defined by @var{pos} and @var{curv}. The variable
+## Draw rectangular patch defined by @var{pos} and @var{curv}.  The variable
 ## @code{@var{pos}(1 : 2)} defines the lower left-hand corner of the patch
-## and @code{@var{pos}(3 : 4)} its width and height. By default the value of
+## and @code{@var{pos}(3 : 4)} its width and height.  By default the value of
 ## @var{pos} is @var{[0, 0, 1, 1]}.
 ##
 ## The variable @var{curv} defines the curvature of the sides of the rectangle
 ## and it can be a scalar or two-element vector with values between 0 and 1.
 ## A value of 0 represents no curvature of the side, where as a value of 1
 ## means that the rectangular side is entirely curved into an arc of a circle.
 ## If curvature is a two-element vector, then the first element is the 
 ## curvature along the x-axis of the patch and the second along y-axis.
@@ -39,17 +39,17 @@
 ## If @var{curv} is a scalar, it represents the curvature of the shorter of the
 ## two sides of the rectangle and the curvature of the other side is defined
 ## by
 ##
 ## @example
 ## min (pos (1: 2)) / max (pos (1:2)) * curv
 ## @end example
 ##
-## Other properties are passed to the underlying patch command. If called 
+## Other properties are passed to the underlying patch command.  If called 
 ## with an output argument, @code{rectangle} returns the handle to the 
 ## rectangle.
 ## @end deftypefn
 ## @seealso{patch}
 
 function h = rectangle (varargin)
 
   [hax, varargin] = __plt_get_axis_arg__ ("rectangle", varargin{:});
@@ -197,16 +197,17 @@ endfunction
 function update_props (h, d)
   kids = get (h, "children");
   set (kids, "edgecolor", get (h, "edgecolor"),
        "linewidth", get (h, "linewidth"),
        "linestyle", get (h, "linestyle"),
        "facecolor", get (h, "facecolor"));
 endfunction
 
+
 %!demo
 %! close all
 %! axis equal
 %! rectangle ("Position", [0.05, 0.05, 0.9, 0.9], "Curvature", [0.5, 0.5]);
 
 %!demo
 %! close all
 %! axis equal
